src/main/java/io/jenkins/docker/connector/DockerComputerJNLPConnector.java

```diff
@@ -43,22 +42,14 @@ public class DockerComputerJNLPConnector extends DockerComputerConnector {
     @CheckForNull
     private String user;
 
-    private final JNLPLauncher jnlpLauncher;
-
     @CheckForNull
     private String jenkinsUrl;
 
     @CheckForNull
     private String[] entryPointArguments;
 
-    @Restricted(NoExternalUse.class)
-    public DockerComputerJNLPConnector() {
-        this(new JNLPLauncher(false));
-    }
-
     @DataBoundConstructor
-    public DockerComputerJNLPConnector(JNLPLauncher jnlpLauncher) {
-        this.jnlpLauncher = jnlpLauncher;
+    public DockerComputerJNLPConnector() {
```

Comment:
Incompatible for JCasC users configuring this connector using something like

```yaml
connector:
  jnlp:
    jnlpLauncher: inbound
```

which should now be (I guess) just

```yaml
connector: jnlp
```

Not sure exactly, without test coverage. Possibly though awkward to restore compatibility by adding a getter/setter that are _not_ `@Deprecated`, as in the core PR. This is assuming again that `tunnel` was not actually configured this way.